Makefile

```diff
@@ -0,0 +1,72 @@
+SHELL := /usr/bin/env bash
+
+# build the registry image
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=openshift-sre
+IMAGE_NAME?=osd-operators
+CATALOG_NAMESPACE?=openshift-operator-lifecycle-manager
+DOCKERFILE?=./Dockerfile
+CHANNEL?=production
+GIT_SHA=$(shell git rev-parse HEAD | cut -c1-8)
+
+TEMPLATE_CS=templates/template_osd-operators.CatalogSource.yaml
+DEST_CS=manifests/00_osd-operators.CatalogSource.yaml
+
+SUBSCRIPTIONS=$(shell cat subscriptions.json)
+
+.PHONY: default
+default: build
+
+.PHONY: clean
+clean:
+	rm -rf manifests/
+
+.PHONY: catalogSource
+catalogSource:
+	# create CatalogSource yaml
+	mkdir -p manifests/
+	cp ${TEMPLATE_CS} ${DEST_CS}
+	sed -i "s/#IMAGE_REGISTRY#/${IMAGE_REGISTRY}/g" ${DEST_CS}
```

Comment:
Nit: This pattern will likely serve better in case of some unexpected error:

```shell
sed -e "s/pattern 1//" -e "s/pattern2//" .... ${TEMPLATE_CS} > ${DEST_CS}
```

Benefits include the ability to re-run this without potentially needing `cp --force`, and if one sed fails, we're not left with a partial artifact.

(This suggestion holds for the next target too)